charts/opentelemetry-operator/README.md

```diff
@@ -62,13 +65,11 @@ _See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation
 
 ## Install Chart
 
-The following command installs the chart with the release name my-opentelemetry-operator in the namespace opentelemetry-operator-system.
-
+The following command installs the chart with the release name `my-opentelemetry-operator` in the namespace `opentelemetry-operator-system`.
+You don't need to provide `--namespace` option here.
```

Comment:
I'd move [the note](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/52/files/7f82e64f0ae10f0e78baebfbe0eaef4d2dc5f000..b4eae4235d6f8ce988476039f95fd9fb9c9787b6#diff-c9849889db5d4f8fc6aa865982eb9bece6cd4bb31fa6286972d5327413a00955R7-R9) here and say that `--namespace` helm argument doesn't have any effect on where the kubernetes objects will be installed. User still can set `--namespace` to put helm release info in any other namespace, if not specified it'll be `default` which is fine I guess.

Reply:
I will rephrase the README.